docs/resources/Guild.md

```diff
@@ -280,18 +280,19 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
+| Field                         | Type                                            | Description                                                                                                                            |
+| ----------------------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| user?                         | [user](#DOCS_RESOURCES_USER/user-object) object | the user this guild member represents                                                                                                  |
+| nick?                         | ?string                                         | this users guild nickname                                                                                                              |
+| avatar?                       | ?string                                         | the member's [guild avatar hash](#DOCS_REFERENCE/image-formatting)                                                                     |
+| roles                         | array of snowflakes                             | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                                       |
+| joined_at                     | ISO8601 timestamp                               | when the user joined the guild                                                                                                         |
+| premium_since?                | ?ISO8601 timestamp                              | when the user started [boosting](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-) the guild                |
+| deaf                          | boolean                                         | whether the user is deafened in voice channels                                                                                         |
+| mute                          | boolean                                         | whether the user is muted in voice channels                                                                                            |
+| pending?                      | boolean                                         | whether the user has not yet passed the guild's [Membership Screening](#DOCS_RESOURCES_GUILD/membership-screening-object) requirements |
+| permissions?                  | string                                          | total permissions of the member in the channel, including overwrites, returned when in the interaction object                          |
+| communication_disabled_until? | ?ISO8601 timestamp                              | when the user's timeout will expire and the user's interaction in the guild is re-enabled, null if the user is not timed out           |
```

Comment:
Maybe we should refer to the FAQ page on Discord Support for how time outs work.
Since line `290` does this for how premium works, we could consider this as well:

https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ

Reply:
this is a great idea 👍